app/src/test/java/mozilla/lockbox/presenter/ItemDetailPresenterTest.kt

```diff
@@ -23,10 +23,14 @@ class ItemDetailPresenterTest {
     class FakeView : ItemDetailView {
         override var itemId: String? = null
         var item: ItemDetailViewModel? = null
+        val tapStub:  PublishSubject<Unit> = PublishSubject.create<Unit>()
 
         override fun updateItem(item: ItemDetailViewModel) {
             this.item = item
         }
+
+        override val hostnameClicks: Observable<Unit>
+            get() = tapStub
```

Comment:
The test compiles. Does it test anything? i.e. please keep the tests the testing the new code, not just the old code.